src/test/compile-fail/issue-13033.rs

```diff
@@ -16,7 +16,9 @@ struct Baz;
 
 impl Foo for Baz {
     fn bar(&mut self, other: &Foo) {}
-    //~^ ERROR method `bar` has an incompatible type for trait: values differ in mutability [E0053]
```

Comment:
See the error message above ^. It is now a label in new-skool and a single note in old-skool.